ui/src/virtualconsole/vcmatrix.cpp

```diff
@@ -284,9 +313,21 @@ void VCMatrix::slotStartColorChanged(QRgb color)
         matrix->updateColorDelta();
 }
 
+void VCMatrix::slotSetEndColor(QColor color)
```

Comment:
@Itay-Lifshitz I'm still not convinced that we need to duplicate start/end color in the widget.
They are stored in the attached RGBMatrix function so let's retrieve them from there.
In case no function is attached, just return QColor()